docs/client_guide/spark_client_guide.md

```diff
@@ -149,4 +149,20 @@ Other configuration:
 |---|---|---|
 |spark.rss.access.timeout.ms|10000|The timeout to access Uniffle coordinator|
 |spark.rss.client.access.retry.interval.ms|20000|The interval between retries fallback to SortShuffleManager|
-|spark.rss.client.access.retry.times|0|The number of retries fallback to SortShuffleManager|
\ No newline at end of file
+|spark.rss.client.access.retry.times|0|The number of retries fallback to SortShuffleManager|
+
+### Partition reassign in one shuffle attempt
+
+To achieve better task stability, partition's reassign mechanism that requests the new replacement shuffleServers is introduced to overcome server instability 
```

Comment:
I think this will be better:

To achieve better task stability, a partition reassignment mechanism has been introduced, which requests new replacement shuffle servers to overcome server instability caused by unhealthy conditions or high memory pressure in a single shuffle attempt. At the present stage, this feature is not compatible with stage retry and multiple replica mechanisms (additional testing is required).

Reply:
I'm glad to accept your suggestion.